torch/onnx/_exporter_states.py

```diff
@@ -31,3 +32,14 @@ def __init__(self, params_dict, env, cur_node, onnx_block):
         self.cur_node: _C.Node = cur_node
         # Current onnx block that converted nodes are being appended to.
         self.onnx_block: _C.Block = onnx_block
+
+
+class RuntimeTypeCheckState(enum.Enum):
+    """Runtime type check state."""
+
+    # Runtime type checking is disabled.
+    DISABLED = 0
+    # Runtime type checking is enabled.
+    ENABLED = 1
```

Comment:
nit: I would recommend swap the order of ENABLED (should we rename it ERRORS?) and WARNINGS_ONLY (WARNINGS?)

Reply:
I think DISABLED=0 makes sense to be the first one (number 0). WDYT?